opm/simulators/wells/MultisegmentWell_impl.hpp

```diff
@@ -1493,20 +1510,28 @@ namespace Opm
             }
             case SURFACE_RATE:
             {
-                // finding if it is a single phase control or combined phase control
```

Comment:
Here, there should be much less code changing here, while some indentation are changed by accident which makes it looks like much more codes are changed.